src/common/file_utils.rs

```diff
@@ -0,0 +1,29 @@
+use crate::common::error::RustBertError;
+use cached_path::Cache;
+use lazy_static::lazy_static;
+use std::env;
+use std::path::PathBuf;
+
+lazy_static! {
+    pub static ref CACHE: Cache = Cache::builder()
+        .dir(_get_cache_directory())
+        .build()
+        .unwrap();
+}
+
+fn _get_cache_directory() -> PathBuf {
+    let home = match env::var("RUSTBERT_CACHE") {
+        Ok(value) => PathBuf::from(value),
+        Err(_) => {
+            let mut home = dirs::home_dir().unwrap();
+            home.push(".cache");
+            home.push(".rustbert");
+            home
+        }
+    };
+    home
+}
+
+pub fn cached_path(resource: &str) -> Result<PathBuf, RustBertError> {
```

Comment:
Could you please wrap the cached path functionality directly in https://github.com/guillaume-be/rust-bert/blob/a52279e24d879db2a4b79833ac409cecd6b3f2ff/src/common/resources.rs#L186 ?